src/Uuid.elm

```diff
@@ -26,25 +26,26 @@ Uuids can be generated either by parsing them from the canonical string represen
 (see fromString) or by generating them. If you are unfamiliar with random number generation
```

Comment:
Above, there are two references to the fact that Elm is not PCG – which is not true anymore, it could be good to update it:

```
using Max Goldsteins Random.PCG pseudo-random generator library
```

```
While Random.PCG is a definite improvement over Elms native RNG, depending
 on your use case the randomness provided may not be enough.
```